src/gradata/enhancements/behavioral_extractor.py

```
@@ -0,0 +1,478 @@
+    return ArchetypeMatch(
+        Archetype.REPLACEMENT_WORD, 0.40,
+        {"old_words": sorted(removed_words)[:3],
+         "new_words": sorted(added_words)[:3]}
+    )
+
+
+# ---------------------------------------------------------------------------
+# Template Generation
+# ---------------------------------------------------------------------------
+
+def generate_instruction(match: ArchetypeMatch, category: str = "") -> str:
```

Ruff (0.15.9)

[warning] Too many return statements (19 > 6)

(PLR0911)